tap_mailchimp/schemas/list_members.json

```diff
@@ -210,6 +269,63 @@
         "null",
         "string"
       ]
+    },
+    "marketing_permissions": {
+      "type": [
+        "object"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "marketing_permission_id": {
+          "type": [
+            "string"
```

Comment:
Does this force `marketing_permission_id` to be string?

Reply:
Added “null” as the fallback option.